wfl/Models/EntityStatus.cs

```diff
@@ -10,7 +10,8 @@ public class EntityStatus
         public enum Status
```

Comment:
I don't foresee this status being a generic thing. So I would name it specific to orders. I would be curious to ear others opinions on this but what do you guys think of defining the enum inside the FoodOrder vs in it's own top level file?